src/ScheduleService.Models.Dto/Requests/Group/FindGroupsRequest.cs

```diff
@@ -0,0 +1,5 @@
+﻿namespace LT.DigitalOffice.ScheduleService.Models.Dto.Requests.Group;
+
+public class FindGroupsRequest
```

Comment:
rename to GetGroupsFilter and inherit from BaseFindFilter

Reply:
+